File: two-strings-are-anagrams.py
# !/usr/bin/env python
# -*- coding: utf-8 -*-
# @author: Rancho (rancho941110@gmail.com)
# @date  : 2016-05-08 20:48:33
#

class Solution:
    """
    @param s: The first string
    @param b: The second string
    @return true or false
    """
    def anagram(self, s, t):
        # write your code here
        ss = [ i for i in s]
        tt = [ i for i in t]
        ss.sort()
        tt.sort()
        return "".join(ss) == "".join(tt)

File: two-strings-are-anagrams.cpp
/*
 * @author: Rancho (rancho941110@gmail.com)
 * @date  : 2016-05-08 20:41:43
 * test two string whether only has different alpha order
 * such as: "abcd", "dcab"
 */
class Solution {
public:
    /**
     * @param s: The first string
     * @param b: The second string
     * @return true or false
     */
    bool anagram(string s, string t) {
        // write your code here
        sort(s.begin(), s.end());
        sort(t.begin(), s.end());
        return s == t;
    }
};
